src/resolve/resolved-dns-packet.c

```diff
@@ -2269,6 +2269,9 @@ int dns_packet_is_reply_for(DnsPacket *p, const DnsResourceKey *key) {
         if (r < 0)
                 return r;
 
+        if (!p->question)
```

Comment:
ok. `$ git grep 'if.*== NULL'` brings up about 229 mostly in udev, and many of them are probably false positives.